testmon/pytest_testmon.py

```diff
@@ -150,6 +150,10 @@ def __init__(self, config, testmon_data):
         self.f_to_ignore = self.testmon_data.stable_files
         if self.config.getoption('tlf'):
             self.f_to_ignore -= self.testmon_data.f_last_failed
+        self.has_possibly_filtering_args = any(
+            x for x in config._origargs
```

Comment:
Not a public attribute: https://github.com/pytest-dev/pytest/issues/5106

Reply:
how about has_k_m_args , or has_filtering_args?

Reply:
`has_filtering_args` would be ok, but it is not given that they actually filter anythings - that's why I had the longer name I guess.